deploy/crds/core_v1_storagecluster_crd.yaml

```diff
@@ -1146,6 +1146,2272 @@ spec:
                             type: boolean
                             description: Flag indicating whether AlertManager needs to be enabled and deployed
                               by the Storage operator.
+                      replicas:
```

Comment:
is it necessary to have all of those specs under prometheus? did we confirm that?

Reply:
A lot of lines are from ```storage``` and ```volumes``` spec. I confirmed with @haibinxie we'll keep everything under ```storage``` to allow more customization options.